src/common/copilot/assets/scripts/copilot.js

```diff
@@ -487,13 +461,21 @@
     }
         case "selectedCodeInfo": {
             const chatInputLabel = document.getElementById("input-label-id");
-            if (message.value.start == message.value.end && message.value.selectedCode.length == 0) {
+            selectedCode = message.value.selectedCode;
+            if (message.value.start == message.value.end && selectedCode.length == 0) {
                 chatInputLabel.classList.add("hide");
                 break;
             }
             chatInputLabel.classList.remove("hide");
             chatInputLabel.innerText = `Lines: ${message.value.start + 1} - ${message.value.end + 1} selected`;
+            break;
+        }
+        case "explainCode": {
+            selectedCode = message.value.selectedCode;
+            const explainPrompt = 'Explain the following code:';
```

Comment:
Best practices. Should be read from a constant.

Reply:
Will be moving the strings out, currently there is no separate constant file for webview script.